systemd/osrt-docker-publisher.service

```diff
@@ -0,0 +1,10 @@
+[Unit]
+Description=openSUSE Release Tools: Docker image publisher
+
+[Service]
+User=opensuse-relmgr-bot
+EnvironmenFile=~/.config/osrt-docker_publisher
```

Comment:
Would seem to make more sense in in sysconfig dir. Similar to:

https://github.com/openSUSE/osc-plugin-factory/blob/3bd6833f89583dea77285a1d2da8c39071e6aa24/systemd/osrt-leaper-manager%40.service#L7

Can also include the base config in `config` dir to have it installed there.

Reply:
No, the file contains a password which must only be readable by that user.